http/server.ts

```diff
@@ -345,20 +367,23 @@ export class Server {
     httpConn: Deno.HttpConn,
     connInfo: ConnInfo,
   ): Promise<void> {
+    let response: Response;
     try {
       // Handle the request event, generating a response.
-      const response = await this.#handler(
+      response = await this.#handler(
         requestEvent.request,
         connInfo,
       );
+    } catch (error) {
+      // Invoke onError handler when request handler throws.
+      response = await this.#onError(error as Error);
```

Comment:
What should we do if this throws? :upside_down_face:

Reply:
then the server should crash